codec-http/src/test/java/io/netty/handler/codec/http/cookie/ClientCookieDecoderTest.java

```diff
@@ -30,26 +30,19 @@
 public class ClientCookieDecoderTest {
     @Test
     public void testDecodingSingleCookieV0() {
-        String cookieString = "myCookie=myValue;expires=XXX;path=/apathsomewhere;domain=.adomainsomewhere;secure;";
-        cookieString = cookieString.replace("XXX",
-                HttpHeaderDateFormatter.format(new Date(System.currentTimeMillis() + 50000)));
+        String cookieString =
+                "myCookie=myValue;expires="
+                        + DateFormatter.format(new Date(System.currentTimeMillis() + 50000))
+                        + ";path=/apathsomewhere;domain=.adomainsomewhere;secure;";
 
         Cookie cookie = ClientCookieDecoder.STRICT.decode(cookieString);
         assertNotNull(cookie);
         assertEquals("myValue", cookie.value());
         assertEquals(".adomainsomewhere", cookie.domain());
-
-        boolean fail = true;
-        for (int i = 40; i <= 60; i++) {
-            if (cookie.maxAge() == i) {
-                fail = false;
-                break;
-            }
-        }
-        if (fail) {
-            fail("expected: 50, actual: " + cookie.maxAge());
-        }
-
+        assertNotEquals("maxAge should be defined when parsing cookie " + cookieString,
+                Long.MIN_VALUE, cookie.maxAge());
+        assertTrue("maxAge should be about 50ms when parsing cookie " + cookieString,
+                cookie.maxAge() >= 40 && cookie.maxAge() <= 60);
```

Comment:
@normanmaurer @Scottmitch This test was the one that failed on CI. I cleaned it up and improved error messages so we could get more information if it fails again.